src/2d/valout.f90

```diff
@@ -382,15 +384,24 @@ subroutine valout(level_begin, level_end, time, num_eqn, num_aux)
              status='old', action='write', position='append')
     end if
     
-    timing_line = "(e16.6"
+    timing_line = "(e16.6, ', ', e16.6, ', ', e16.6,"
     do level=1, mxnest
         timing_substr = "', ', e16.6, ', ', e16.6, ', ', e16.6"
         timing_line = trim(timing_line) // timing_substr
     end do
     timing_line = trim(timing_line) // ")"
 
-    call cpu_time(t_CPU_overall)
-    write(out_unit, timing_line) t_CPU_overall, &
+    if (time == t0) then
```

Comment:
Can this be set in the `amr_module` or something similar?  I am worried that the comparison might not work or that this will get mucked up on a restart.